packages/web/src/pages/search-page/search-results/AllResults.tsx

```diff
@@ -50,13 +64,152 @@ export const AllResults = ({ handleSearchTab }: AllResultsProps) => {
 
   if (showNoResultsTile) return <NoResultsTile />
 
+  const SearchResultItem = ({
```

Comment:
you should not define components in the render method, these need to be stable between rerenders, otherwise react constantly makes new component instances